.github/workflows/ci.yml

```diff
@@ -4,22 +4,21 @@ on:
   pull_request: {}
 jobs:
   run:
-    runs-on: macos-11
+    runs-on: ${{ matrix.runsOn }}
     name: Xcode ${{ matrix.xcode }}
+    env: 
+      DEVELOPER_DIR: "/Applications/Xcode_${{ matrix.xcode }}.app/Contents/Developer"
+    timeout-minutes: 10
     strategy:
+      fail-fast: false
       matrix:
-        xcode: ["12.5.1", "13.0"]
+        include:
+          - xcode: "13.0"
+            runsOn: macOS-11
+          - xcode: "12.5.1"
+            runsOn: macOS-11
     steps:
     - uses: actions/checkout@master
-    - name: Set Xcode
-      run: |
-        echo "Available Xcode versions:"
-        ls /Applications | grep Xcode
-        echo "Choosing Xcode_${{ matrix.xcode }}.app"
-        sudo xcode-select -s /Applications/Xcode_${{ matrix.xcode }}.app
```

Comment:
Nice cleanup. I see the `env. DEVELOPER_DIR` is a better way to do this nowadays